pkg/scheduler/cache/cache.go

```diff
@@ -115,7 +114,7 @@ type SchedulerCache struct {
 	quotaInformer              infov1.ResourceQuotaInformer
 	csiNodeInformer            storagev1.CSINodeInformer
 	csiDriverInformer          storagev1.CSIDriverInformer
-	csiStorageCapacityInformer storagev1beta1.CSIStorageCapacityInformer
```

Comment:
If volcano deployed in old version k8s, is v1 api supported in old version, we should clarify when v1beta1 is introduced and removed to make the api compatibility.

Reply:
Hmm, I am agree with you. But I think that a new issue is to discuss this.